test/zdtm.py

```diff
@@ -2672,6 +2672,13 @@ def clean_stuff(opts):
 cp.add_argument("what", choices=['nsroot'])
 
 opts = vars(p.parse_args())
+
+try:
+    if os.environ["ZDTM_IGNORE_TAINT"] == "1":
```

Comment:
if os.getenv("ZDTM_IGNORE_TAINT") == "1":
  opts["ignore_taint"] = True

In this case, you will not need to handle the KeyError exception.